src/haconiwa/task/cli.py

```
@@ -118,5 +118,52 @@ def prune(
+            worktree_path=worktree_path,
+            dry_run=dry_run
+        )
+        
+        if not dry_run:
+            console.print(f"✅ Task successfully submitted: [bold green]{title}[/]")
+            console.print(f"📁 Worktree: [cyan]{result.get('worktree_path')}[/]")
+            console.print(f"👤 Assignee: [yellow]{assignee}[/]")
+            console.print(f"🌿 Branch: [blue]{branch}[/]")
+            
+            # Show suggestion to monitor
+            console.print(f"\n💡 To monitor progress, run:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)